driver/razerkbd_driver.h

```diff
@@ -77,6 +77,7 @@
 #define USB_DEVICE_ID_RAZER_CYNOSA_V2 0x025E
 #define USB_DEVICE_ID_RAZER_BOOK_2020 0x026A
 #define USB_DEVICE_ID_RAZER_BLADE_15_ADV_EARLY_2021 0x026D
+#define USB_DEVICE_ID_RAZER_BLACKWIDOW_V3 0x024E
```

Comment:
Move this just below `USB_DEVICE_ID_RAZER_BLADE_STUDIO_EDITION_2019` to keep the list sorted

Reply:
Moved.